gammapy/datasets/core.py

```diff
@@ -21,9 +21,9 @@ class Dataset(abc.ABC):
     """Dataset abstract base class.
     For now, see existing examples of type of datasets:
 
-    - `gammapy.datasets.MapDataset`
-    - `gammapy.datasets.SpectrumDataset`
-    - `gammapy.datasets.FluxPointsDataset`
+    - `gammapy.datasets.MapDataset`.
+    - `gammapy.datasets.SpectrumDataset`.
+    - `gammapy.datasets.FluxPointsDataset`.
```

Comment:
In English when we have a list in this format, we do not want to place a full stop at the end